packages/lab/src/dialog/Dialog.tsx

```diff
@@ -36,14 +38,22 @@ export interface DialogProps extends HTMLAttributes<HTMLDivElement> {
    */
   onOpenChange?: (open: boolean) => void;
   /**
-   * Status indicator
+   * The status of the Dialog
```

Comment:
would be helpful to note that what the statuses are for users to  quickly grab and use

Reply:
These are noted below where the prop is passed into the component where they are picked up by the JS Docs, so i think not sure they're needed here as well

Reply:
The types are declared below when passed into the component, this is also where they're picked up by the JSDocs.
It's typed here as `Validation Status` i could break this down if you think it is more user friendly but I think it is fine